data/mainnet/tokens.js

```diff
@@ -83,6 +83,26 @@ module.exports = {
     cmcRewardTokenSymbols: ['iFARM', 'DON', 'WETH'],
     disableAutoSwap: true,
   },
+  crvIbEUR: {
+    isNew: true,
+    chain: CHAINS_ID.ETH_MAINNET,
+    category: VAULT_CATEGORIES_IDS.STABLECOINS,
+    displayName: 'CRV:ibEUR',
+    apyIconUrls: ['./icons/curve-ibeur.png', './icons/cvx.png', './icons/spell.png'], // what is spell? 
+    apyTokenSymbols: ['CRV', 'CVX', 'SPELL'], // what is spell? 
+    logoUrl: './icons/curve-ibeur.png',
+    tokenAddress: addresses.V2.convex_ibEUR.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.V2.convex_ibEUR.NewVault,
+    priceFunction: { type: GET_PRICE_TYPES.COINGECKO_CONTRACT, params: [addresses.ibEUR] }, // how can I find out the type?
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.CONVEX,
+        params: ['ibEUR', convexProfitSharingFactor], // how can I find out params?
+      },
+    ],
+    cmcRewardTokenSymbols: ['FARM', 'CVX', 'SPELL', 'CRV'], // what is cmc?
```

Comment:
cmc = CoinMarketCap, this field should help provide data to them in the format they want. Please change FARM to iFARM, remove SPELL and add rK3PR. Please also change FARM at the MIM and EURT entries to iFARM, I see there has been a mistake there.